release_tester/selenium_ui_test/test_suites/base_test_suite.py

```diff
@@ -46,26 +48,24 @@ def __init__(self, child_classes=[]):
         if self.new_version:
             versions.append(self.new_version)
         if hasattr(self, "generate_custom_suite_name"):
-            #pylint: disable=no-member
+            # pylint: disable=no-member
             self.suite_name = self.generate_custom_suite_name()
         if self.use_subsuite:
             self.sub_suite_name = self.__doc__ if self.__doc__ else self.__class__.__name__
         else:
             self.sub_suite_name = None
         self.test_suite_context = AllureTestSuiteContext(
-                properties=RunProperties(self.enterprise,
-                                         self.enc_at_rest,
-                                         self.ssl),
-                versions=versions,
-                parent_test_suite_name=None if not self.parent_test_suite_name else self.parent_test_suite_name,
-                auto_generate_parent_test_suite_name=True
-                if not hasattr(self, "auto_generate_parent_test_suite_name")
-                else self.auto_generate_parent_test_suite_name,
-                suite_name=None if not self.suite_name else self.suite_name,
-                sub_suite_name=None if not self.sub_suite_name else self.sub_suite_name,
-                runner_type=None if not self.runner_type else self.runner_type,
-                installer_type=None if not self.installer_type else self.installer_type,
-            )
+            properties=RunProperties(self.enterprise, self.enc_at_rest, self.ssl),
+            versions=versions,
+            parent_test_suite_name=None if not self.parent_test_suite_name else self.parent_test_suite_name,
+            auto_generate_parent_test_suite_name=True
+            if not hasattr(self, "auto_generate_parent_test_suite_name")
```

Comment:
its doing nothing on if, but only on else? why not reverse condition and have no else?

Reply:
Those 3 lines are one expression
```
            auto_generate_parent_test_suite_name=True
            if not hasattr(self, "auto_generate_parent_test_suite_name")
            else self.auto_generate_parent_test_suite_name
```